src/pages/chapter/forms/index.tsx

```diff
@@ -1,7 +1,108 @@
-import { Flex } from "@chakra-ui/react";
+import { AddIcon } from "@chakra-ui/icons";
+import { Heading, Box, Button, HStack, Text } from "@chakra-ui/react";
+import FormCard from "src/components/FormCard";
+import { FormType } from "src/utils/types";
 
 function ChapterFormsPage() {
-  return <Flex>Chapter Forms Page</Flex>;
+  const forms = [
+    {
+      _id: `1`,
+      name: "Application Form 1",
+      type: FormType.APPLICATION,
+      updatedAt: new Date(),
+    },
+    {
+      _id: `2`,
+      name: "Application Form 2",
+      type: FormType.APPLICATION,
+      updatedAt: new Date(),
+    },
+    {
+      _id: `3`,
+      name: "Feedback Form 1",
+      type: FormType.FEEDBACK,
+      updatedAt: new Date(),
+    },
+    {
+      _id: `4`,
+      name: "Application Form 1",
+      type: FormType.APPLICATION,
+      updatedAt: new Date(),
+    },
+    {
+      _id: `5`,
+      name: "Application Form 2",
+      type: FormType.APPLICATION,
+      updatedAt: new Date(),
+    },
+    {
+      _id: `6`,
+      name: "Feedback Form 1",
+      type: FormType.FEEDBACK,
+      updatedAt: new Date(),
+    },
+  ];
+  return (
+    <Box height="100%" backgroundColor="#EBEEF1" overflow="auto">
+      <Box
+        display="table"
+        boxSizing="border-box"
+        padding="40px 40px 40px 40px"
+        justifyContent="center"
+        alignItems="center"
+        margin="auto"
+        align="center"
+      >
+        <Box width="1000px">
+          <Heading
+            fontSize={{ base: "x-large", md: "xx-large" }}
+            marginBottom="30px"
+            align="left"
+          >
+            My Forms
+          </Heading>
+        </Box>
+
+        <Box
+          flexDirection="column"
+          width="1000px"
+          height="550px"
+          padding="30px"
+          backgroundColor="white"
+          border="1px"
+          borderColor="#657788"
+          borderRadius="15px"
+        >
+          <Box align="right" marginBottom="30px">
+            <Button colorScheme="blue">
+              <HStack>
+                <AddIcon />
+                <Text>Create New Form</Text>
+              </HStack>
+            </Button>
+          </Box>
+          <Box
+            flexDirection="column"
+            width="900px"
+            height="400px"
```

Comment:
Like before, do not use fixed dimensions here.  Use padding and margin to resize the container relative to the parent container.